pom.xml

```diff
@@ -1,168 +1,185 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.5</version>
+        <version>2.9</version>
     </parent>
 
-    <artifactId>run-selector</artifactId>
+    <artifactId>run-selector-api</artifactId>
     <packaging>hpi</packaging>
-    <name>Run Selector Plugin</name>
-    <version>1.0-SNAPSHOT</version>
-    <url>http://wiki.jenkins-ci.org/display/JENKINS/Run+Selector+Plugin</url>
+    <name>Run Selector API Plugin</name>
```

Comment:
WDYT if we call it Run Selector **API** Plugin, because it's not a standalone plugin.